lib/constable_web/templates/announcement/index.html.eex

```diff
@@ -12,22 +12,36 @@
     </div>
   </nav>
 
-  <p class="interests-message container">
-    <%= dngettext("interest-count", "You're subscribed to 1 interest.", "You're subscribed to %{count} interests.", interest_count_for(@current_user)) %>
-    <%= link to: interest_path(@conn, :index), data: [role: "view-all-interests"] do %>
-      <%= gettext "Manage your interests." %>
-    <% end %>
-  </p>
+  <%= if !@show_all do %>
+    <p class="interests-message <%= class_for("mine", @conn) %>">
+      <%= link to: interest_path(@conn, :index), data: [role: "view-all-interests"] do %>
+        <span class="subscribed-to">
```

Comment:
Maybe unnest this and put that class on the `a` and remove the `span`? Not sure if that makes the styling janky.

Reply:
Oh wait I see why you did this now. The edit text is a separate thing. Disregard my earlier comment :)